lib/nql.js

```diff
@@ -33,7 +33,7 @@ module.exports = (queryString, options) => {
     };
 
     // Use MongoKnex to apply the query to a query builder object
-    api.querySQL = qb => mongoKnex(qb, api.parse());
+    api.querySQL = qb => mongoKnex(qb, api.parse(), options);
```

Comment:
`options` - this is where `filters` and `relations config` would come through?

Reply:
yes

Reply:
not filters, only the config

Reply:
How would filters come in then?

Reply:
https://github.com/NexesJS/NQL/blob/master/lib/nql.js#L13

Reply:
NQL supports a single filter currently.

Reply:
The PR is about: bumping mongo-knex and forwarding the options. The options are needed to use relation queries.

Reply:
Sure, makes sense. Just asking :) :shipit: